test/ansible-container/README.md

```diff
@@ -0,0 +1,4 @@
+# WARNING
+Do not change anything in this directory!
```

Comment:
There should be a "why" here :) Maybe `Do not change anything in this directory! It is auto-generated and all changes will be overriden` or similar (whatever the deeper reason is)

Reply:
This was the intention of the next line "See …" and people should go read there 🤔
but it's a good input I'll add it if I don't remove the whole directory 😉